File: app/src/main/java/com/p4r4d0x/genreclassifier/rest/StatsRetrofitService.java
package com.p4r4d0x.genreclassifier.rest;

import com.p4r4d0x.genreclassifier.rest.stats.StatsResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface StatsRetrofitService {

    @GET("dev/classifier/stats")
    Call<StatsResponse> userStats(@Query("userId") Long userId);

}

File: app/src/main/java/com/p4r4d0x/edmgenreclassifier/datasource/retrofit/StatsRetrofitService.kt
package com.p4r4d0x.edmgenreclassifier.datasource.retrofit

import com.p4r4d0x.edmgenreclassifier.datasource.dto.UserStatsDTO
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface StatsRetrofitService {
    @GET("dev/classifier/stats")
    fun userStats(@Query("userId") userId: Long?): Call<UserStatsDTO?>?
}